students/JaNiece/contact.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="index.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> My Webpage </title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <img src="images/sm.jpeg" class="logo">
+      <a href="index.html"> Home </a>
+      <a href="life.html"> Learn More </a>
+      <a href="goals.html"> My Goals </a>
+      <a href="contact.html" class= "right"> Contact </a>
+    </nav>
+  </header>
+
+  <p> Email: <a href="mailto:jlevelston@gmail.com"> jlevelston@gmail.com</a> <p>
```

Comment:
🔧 Make sure to close your <p> tag. It can be helpful to validate your HTML in a validator to catch things like this - try the one @nellarro linked to in HTML Resources